.github/workflows/ci.yaml

```diff
@@ -6,20 +6,20 @@ on:
       - master
 
 jobs:
-  lint:
-    name: Lint
+  verify:
+    name: Verify
     runs-on: ubuntu-latest
-    container: golangci/golangci-lint:v1.32.2
-    timeout-minutes: 10
+    container: gcr.io/k8s-staging-releng/releng-ci:latest
```

Comment:
used this image because we maintain that in the k8s sig-release and i don't that contains all the things we need to verify, but we can change to another or even build a custom one.